src/Elmish.WPF/Merge.fs

```diff
@@ -66,10 +66,6 @@ module CollectionTarget =
       Enumerate = ct.Enumerate
       GetCollection = ct.GetCollection >> fOut }
 
-  let map outMapA outMapCollection inMapA =
-    mapA outMapA inMapA
-    >> mapCollection outMapCollection
-
```

Comment:
From https://github.com/elmish/Elmish.WPF/pull/470#discussion_r963220149:

> I am not convinced that the changes in this file are needed yet. For example, `BindingData.OneWaySeq.boxMinorTypes` boxes `'a` while `Helpers.createBinding` boxes `'aCollection`. Can you change this PR so that `aCollection` is boxed in each binding type's `boxMinorTypes` function?

-- @TysonMN 

> When we add the `'t` type parameter to `BindingData`, we are promoting types like `'aCollection` from a "minor" type to a major type. Subsequently (to maintain backwards compatibility) `Helpers.createBinding` boxes one of the _major_ types of `BindingData` (ie, `'t`) so that we don't break existing code.
> 
> In one of the next PRs, we will be adding something like `Helpers.createBindingT` that doesn't box the major type, and then use it in the new `StaticHelper` class.

-- @marner2

Reply:
This is less efficient and convincingly correct for you static code, but what about (just to start) you unbox `'t` in the future PR instead of merging this PR?

Reply:
I think you mean just merging #511 first? I actually split them apart so you can merge that one first.

In the end though, we need this type passed all the way through, otherwise we'll be stuck putting type annotations on the view model properties.

Reply:
@TysonMN I am starting to do (approximately) this in #522.